packages/core/src/command/hl7-sftp-ingestion/psv-converter-schema.ts

```diff
@@ -18,31 +19,55 @@ const PatClassEnum = z
   .optional()
   .transform(val => {
     if (!val) return undefined;
-    //Test data has some common patient class names that we need to map to HL7 codes
     const mapping: Record<string, string> = {
-      OBSERVATION: "O",
+      OBSTETRICS: "B",
+      "COMMERCIAL ACCOUNT": "C",
       EMERGENCY: "E",
       INPATIENT: "I",
+      "NOT APPLICABLE": "N",
       OUTPATIENT: "O",
-      AMBULATORY: "A",
-      BIRTH: "B",
-      CLINIC: "C",
-      NEWBORN: "N",
-      "PRE-ADMISSION": "P",
-      RECURRING: "R",
+      PREADMIT: "P",
+      "RECURRING PATIENT": "R",
       UNKNOWN: "U",
     };
-    return mapping[val.toUpperCase()] ?? val;
-  })
-  .refine(val => !val || ["B", "C", "E", "I", "N", "O", "P", "R", "U"].includes(val), {
-    message: "Patient class must be a valid HL7 code (B, C, E, I, N, O, P, R, U)",
+    const mappedValue = mapping[val.toUpperCase()] ?? val;
+    const validCodes = ["B", "C", "E", "I", "N", "O", "P", "R", "U"];
```

Comment:
@RadmirGari realized we already have these in codebase - see [here](https://github.com/metriport/metriport/blob/18751f9194749f612114e407b69d473d813c3dc2/packages/core/src/command/hl7v2-subscriptions/hl7v2-to-fhir-conversion/adt/mappings.ts#L1)

Let's import those.